proj/pred/templates/pred/submit_seq.html

```diff
@@ -108,8 +108,7 @@ <h2>PRODRES: Protein Domain Reduced Search</h2>
      <tr><td> <h3>Reference</h3> </td></tr>
          {% include "pred/refbib.html" %}
      <tr><td> <h3>Resource provider</h3> </td></tr> 
-     <tr><td>PRODRES is supported by the <a href="https://www.egi.eu/services/cloud-compute/">EGI FedCloud</a> with the VO VO.NBIS.SE for computational resource. </td></tr> 
-     </td></tr>
+       {% include "pred/egi_acknowledgement.html" %}
```

Comment:
```suggestion
     <tr><td> 
       <h3>Resource provider</h3>
       {% include "pred/egi_acknowledgement.html" %}
     </td></tr>
```

Reply:
Good that you noticed this! Fixed in 34c3f77

Reply:
Looks great now!